### ISSUE TITLE
Setting cover-card jekyll theme to xxxxx.github.io doesn't work

### ISSUE THREAD

**Issue by bungabunga** [epidrome/cover-card#13]
I appologise in advance, this is probably not a bug, only my lack of knowledge. it seems that this jekyll theme only works if you set github page like this - [jurezebec.github.io/jurezebec](jurezebec.github.io/jurezebec), and not like this - [jurezebec.github.io](jurezebec.github.io). is it possible to somehow make it work the other way, because then the adress is simpler to remember?

thanks!

**Comment by epidrome** [comment]
i think your question is not related to the theme but with the operation of Github Pages
https://help.github.com/articles/user-organization-and-project-pages/

**Comment by bungabunga** [comment]
what makes me think that this is somehow related to the theme is that if i follow the procedure on your link the cover-card site is online and visible for a minute but later it dissapears, it is replaced with all white site with hidden (white on white) text: 
<username>.github.io    Just a business card, no blog required!

**Comment by epidrome** [comment]
are you using the gh-pages branch in your Github Pages settings? this should be the default, when forking

**Comment by bungabunga** [comment]
when you set your page as username.github.io there's no other option as 'master'. the setting is grayed out,
![scr](https://user-images.githubusercontent.com/2994878/47574920-100a1d00-d941-11e8-9d20-45f7f517d04f.png)

**Comment by epidrome** [comment]
thanks, now i have a better understanding of the issue

here is the explanation why it does not work, Github Pages does not allow serving from gh-pages branch if you are on your user page
https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/

of course, we could do a refactoring of cover-card, but this would be a major one in terms of development workflow, because the master branch is occupied with the source code of the gem-based theme

have you considered to get a nice memorable domain name? it should be better than the generic github.io

**Comment by bungabunga** [comment]
i'll see about that. great that we now know the reason behind this issue. thanks!

**Comment by epidrome** [comment]
thanks for reporting this, i have filled it as a bug and will keep it as an open issue until it is resolved

besides getting a domain name, another solution is to delete (or rename) your `master branch` and then rename your `gh-branch` to `master`, then it should work according to your specification.

hope this helps!

**Comment by bungabunga** [comment]
vow! will try that, thanks.

**Comment by nickgarlis** [comment]
@bungabunga Renaming your repository to `bungabunga.github.io` should solve your issue.

**Comment by bungabunga** [comment]
@nickgarlis nope. and that's the reason for this issue.

**Comment by epidrome** [comment]
@nickgarlis no, it does not work because of "bad chemistry" between Github Pages and theme design, see:
https://github.com/epidrome/cover-card/issues/13#issuecomment-433463399

**Comment by ans1genie** [comment]
Hello,
Any updates about this issue?
